core/services/keystore/keys/workflowencryptionkey/export.go

```diff
@@ -0,0 +1,44 @@
+package workflowencryptionkey
+
+import (
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+
+	"github.com/smartcontractkit/chainlink/v2/core/services/keystore/keys"
+	"github.com/smartcontractkit/chainlink/v2/core/utils"
+)
+
+const keyTypeIdentifier = "Encryption"
+
+func FromEncryptedJSON(keyJSON []byte, password string) (Key, error) {
+	return keys.FromEncryptedJSON(
+		keyTypeIdentifier,
+		keyJSON,
+		password,
+		adulteratedPassword,
+		func(_ keys.EncryptedKeyExport, rawPrivKey []byte) (Key, error) {
+			return Raw(rawPrivKey).Key(), nil
+		},
+	)
+}
+
+func (k Key) ToEncryptedJSON(password string, scryptParams utils.ScryptParams) (export []byte, err error) {
```

Comment:
Can you use exportutils.go for these two methods?

Reply:
hey @pavel-raykov, they are using the `keys.FromEncryptedJSON` and `keys.ToEncryptedJSON` that are ithe methods in `exportutils.go`

Reply:
sorry, I was confused.